src/plugin.js

```diff
@@ -25,10 +31,20 @@ function initPlugin() {
           }
         },
         (err) => {
-          alert('Upload failed');
+          editor.dom.remove(`*[id="tinymce-new-image-${file.name}"]`);
+          editor.getParam('image_conversion_error_handler')();
         }
       );
     }
+
+    reader.onerror = () => {
+      reader.abort()
```

Comment:
Might as well log to the console too, if only to give users a chance to help us find these. WDYT?

Reply:
Seems reasonable.